vega_sim/scenario/common/agents.py

```diff
@@ -1209,7 +1210,7 @@ def __init__(
                 q_lower=self.q_lower,
                 mdp=self.mdp,
                 kappa=self.market_kappa,
-                lmbda=self.market_order_arrival_rate,
+                Lambda=self.market_order_arrival_rate,
```

Comment:
D'oh, meant to change these both to lmbda. We can stick to Lambda for now though :+1:

Reply:
Actually, I've changed my mind - I'll do a separate PR - otherwise this will touch even more files.